org_chart.js

```diff
@@ -1,73 +1,132 @@
-let elon = {
-  name: "Elon Musk",
-  email: "elon.musk@musky.com",
-  position: "Boss",
+"use strict";
+
+function Employee(name, email, position, gender, project){
+  this.name = name;
+  this.email = email;
+  this.position = position;
+  this.gender = gender;
+  if (!project) {
+  this.project = "";
+  } else {
+  this.project = project;
+  }
 }
 
-let new_hires = [
-  {
-    name: "Alan Turing",
-    email: "alan.turing@musky.com",
-    position: "Computer Scientist",
-    project: "Cryptography"
-  },
-  {
-    name: "Grace Hopper",
-    email: "grace.hopper@musky.com",
-    position: "Computer Scientist",
-    project: "Compilers"
-  },
-  {
-    name: "Donald Knuth",
-    email: "donald.knuth@musky.com",
-    position: "Mathematician",
-    project: "Algorithm Analysis"
+Employee.prototype.addSkill = function(skills) {
+  if (!this.skills) {
+    this.skills = [];
+  } else {
+  this.skills = skills;
   }
-]
+}
 
-// assign each a manager
-let [alan, grace, donald] = new_hires
+Employee.prototype.setManager = function(manager) {//not working?
+  this.manager = manager;
+}
 
-alan.manager = elon
-grace.manager = elon
-donald.manager = elon
+Employee.prototype.print = function(){ //working
+  console.log(`${this.name} is a ${this.position} working on ${this.project}.`)
+  if (this.gender === "female") {
+      console.log(`She can be reached at ${this.email}`)
+  } else {
+  console.log(`He can be reached at ${this.email}`)
+  }
+};
 
-// send to training
-if (!alan.skills) {
-  alan.skills = []
-}
-alan.skills.push("Mathematics")
 
-if (!grace.skills) {
-  grace.skills = []
-}
-grace.skills.push("Cobol")
+let elon = new Employee("Elon Musk","elon.musk@musky.com","male","Boss");
+let alan = new Employee("Alan Turing","alan.turing@musky.com","Computer Scientist","male", "Cryptography");
+let grace = new Employee("Grace Hopper","grace.hopper@musky.com","Computer Scientist","female","Compilers");
+let donald = new Employee("Donald Knuth","donald.knuth@musky.com","Mathematician","male","Algorithm Analysis");
+let tim = new Employee("Tim Berners-Lee","tim.berners_lee@musky.com","Computer Science","male","Networks");
 
-if (!donald.skills) {
-  donald.skills = []
-}
-donald.skills.push("Computational Complexity")
+console.log("elon", elon);
+elon.print();
+elon.setManager("nobody");
+console.log("elon", elon);
+
+grace.print();
+
+donald.setManager("Elon Musk");
+alan.setManager("Elon Musk");
+alan.setManager("Elon Musk");
+
+alan.addSkill("Mathematics");
+grace.addSkill("Cobol");
+donald.addSkill("Computational Complexity");
+tim.addSkill("Protocols");
+
```

Comment:
WHere are you calling `.print()`?
You can then further DRY this by making a function that sets the manager, adds the skill, and prints the person